main_2023.py

```diff
@@ -22,6 +22,7 @@
 DETECT_TARGET_DEVICE = 0  # Use "cpu" if no CUDA
 DETECT_TARGET_MODEL_PATH = "tests/model_example/yolov8s_ultralytics_pretrained_default.pt"  # TODO: Update
 DETECT_TARGET_SAVE_PREFIX = "log_comp"
+DETECT_TARGET_ENABLE_HALF_PRECISION = True  # Use False if no CUDA
```

Comment:
When the configuration files and arguments are merged, this will change. We can wait to merge this PR until then.